python/setup.py

```diff
@@ -6,7 +6,7 @@
 package_dir = os.path.dirname(__file__)
 
 # Put all package dependencies here
-requires = ["fire", "numpy"]
+requires = ["fire", "numpy==1.19.0"]
```

Comment:
Is there any reason we force a numpy version?

Reply:
Yes, I'm forcing numpy to a version that still supports Python 3.6. Otherwise, our CI fails to build for Python 3.6.

Reply:
Once we fix the Rust tests, the CI should be fine again.